packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -27,12 +27,12 @@ export type AlertConsumers = typeof AlertConsumers[keyof typeof AlertConsumers];
 export type STATUS_VALUES = 'open' | 'acknowledged' | 'closed' | 'in-progress'; // TODO: remove 'in-progress' after migration to 'acknowledged'
 
 export const mapConsumerToIndexName: Record<AlertConsumers, string | string[]> = {
-  apm: '.alerts-observability-apm',
-  logs: '.alerts-observability.logs',
-  infrastructure: '.alerts-observability.metrics',
+  apm: '.alerts-observability.apm.alerts',
+  logs: '.alerts-observability.logs.alerts',
+  infrastructure: '.alerts-observability.metrics.alerts',
   observability: '.alerts-observability',
```

Comment:
Does it make sense to change it to `.alerts-observability.*.alerts`?

Reply:
This object should be updated to append the space id in the index name as well as reading from the kibana config for observability's usage of custom prefixes. Not sure if these are still happening @peluja1012 ?

@banderror It would be ideal to implement this data structure as a saved object, this way we provide the customer with some way to mutate the index names without having to push out a new release of kibana.

Reply:
As long as we don't index evaluation docs the result would be the same, but the change also couldn't hurt while we're at it.